tests/fips/mozilla_nss/firefox_nss.pm

```diff
@@ -34,43 +28,44 @@ sub run() {
     # launch firefox first to generate its default profile
     x11_start_program("firefox");
     assert_screen("firefox-launch", 90);    # launch firefox need time
-    quit_firefox;
+    send_key "ctrl-q", 1;
 
```

Comment:
just add a wait_still_screen here instead of wait_idle and stop adding wait_idle flags to send_key. I will remove this parameter soon enough